src/style-spec/expression/definitions/within.js

```diff
@@ -7,22 +7,30 @@ import type {Expression} from '../expression';
 import type ParsingContext from '../parsing_context';
 import type EvaluationContext from '../evaluation_context';
 import type {GeoJSON, GeoJSONPolygon, GeoJSONMultiPolygon} from '@mapbox/geojson-types';
-import MercatorCoordinate from '../../../geo/mercator_coordinate';
-import EXTENT from '../../../data/extent';
 import Point from '@mapbox/point-geometry';
 import type {CanonicalTileID} from '../../../source/tile_id';
```

Comment:
Do we care that it's also importing types from outside the style-spec?

Reply:
no, coz they are completely ignored by module resolution, its just used by flow to typecheck.